checkov/common/checks/object_registry.py

```diff
@@ -41,7 +41,7 @@ def _scan_array(
                         entity_type,
                         results,
                         scanned_file,
-                        skip_info,
+                        skip_info[0],
```

Comment:
What's the reason for only selecting the first element? what if it's an empty list?

Reply:
in the `_scan()` method I make sure, it has at least one entry `[{}]` so it works as before